src/m365/teams/commands/chat/chat-message-list.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Message } from '../../Message';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  chatId: string;
+}
+
+class TeamsChatMessageListCommand extends GraphItemsListCommand<Message> {
+  public get name(): string {
+    return commands.CHAT_MESSAGE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all messages from a chat';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'summary', 'body'];
```

Comment:
We either need to trim the body to a fixed length or remove it altogether as for longer messages, it makes the terminal output unreadable:

![image](https://user-images.githubusercontent.com/11164679/147595782-2e00c8a0-b6c1-408e-bf26-879e85e5b73f.png)

Also, for me, the summary shows `null`. Have you ever seen it contain anything else? If not, we should consider removing it from the default output as it won't add any value.

Reply:
ok, I've updated the code. I added an extra `shortBody` property *only* when outputting in `text` mode. I reused truncation code from a yammer message-list file. (I duplicated the code, I could also move the code to a Util function, but then I would need to edit the yammer command code as well)

`summary` It does show `null` indeed, I've removed it.